packages/lesswrong/components/hooks/useDialogueMatchmaking.ts

```diff
@@ -43,6 +46,15 @@ export const useDialogueMatchmaking = <T extends MatchmakingProps>(props: T): Us
     }
   `, { skip: skipByDefault || !getMatchedUsers });
 
+  const recommendedUsersQueryResult = useQuery(gql`
+  query GetDialogueMatchedUsers {
+    GetDialogueRecommendedUsers {
+      _id
+      displayName
+    }
+  }
+  `, { skip: !getRecommendedUsers || !dialogueMatchmakingEnabled.get() });
```

Comment:
use `skipByDefault` rather than `dialogueMatchmakingEnabled.get()`, which also checks that there's a `currentUser` (i.e. this is a logged-in user).  I don't think we want to be making this query for logged out users?

Reply:
we do want it to be dependent on that public db variable, though?